### PR TITLE
Quit function

### LINKED ISSUE #63: Implement the exit game button in the game menu

### PR DESCRIPTION
**Pull request by Mart500**
closes #63 
Added function in canvasScript and linked it to the quit button in the option menu
Will not quit in editor, will only quit in build.

### CODE REVIEW

File: Assets/Code/Scripts/GUI/canvasScript.cs [hunk 1]
```diff
@@ -309,4 +309,9 @@ private string FormatResourceChangeLabel(int changeAmount)
         string sign = (changeAmount >= 0) ? "+" : "";
         return "(" + sign + changeAmount.ToString() + ")";
     }
+
```
Comment (RobinStephenson) [inline Assets/Code/Scripts/GUI/canvasScript.cs:334]: needs a note saying added by JBT

**Review by RobinStephenson — CHANGES_REQUESTED** [review]

**Comment by Mart500** [comment]
Thanks for double checking. @RobinStephenson Fixed now.

**Comment by RobinStephenson** [comment]
@Mart500  do you want to resolve the conflicts as well

**Review by RobinStephenson — APPROVED** [review]

**Comment by Mikeywalsh** [comment]
@RobinStephenson sure, one sec

**Comment by Mart500** [comment]
@Mikeywalsh You ok with merging then?

**Pull request merged**